tox.ini

```diff
@@ -37,6 +37,14 @@ commands=
 setenv=
     NUMBA_DISABLE_JIT=1
 
+[testenv:data]
+deps=
+    -rtest_requirements.txt
+    -rdata_requirements.txt
```

Comment:
If we inline `data_requirements.txt` here, the tox environment will be automatically re-created when they change. What do you think?

Reply:
Sounds good! 👍